sdk/python/kfp/dsl/_container_op.py

```diff
@@ -630,6 +630,95 @@ def inputs(self):
         return _pipeline_param.extract_pipelineparams_from_any(self)
 
 
+class InitContainer(Container):
```

Comment:
InitContainer is exactly same as Sidecar if my understanding is correct. Could you use the same class to represent both of them to avoid duplication? Maybe call it UserContainer?

Reply:
done. also i need to keep the sidecar interface to avoid breaking change but switch and reuse user container as implementation.